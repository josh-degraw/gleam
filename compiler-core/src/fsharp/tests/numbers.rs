use crate::assert_fsharp;

#[test]
fn simple_variable_assignment() {
    assert_fsharp!(
        r#"
pub fn main() {
  let a = 3
}
"#
    );
}
