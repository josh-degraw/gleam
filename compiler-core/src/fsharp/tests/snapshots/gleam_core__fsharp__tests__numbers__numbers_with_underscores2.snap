---
source: compiler-core/src/fsharp/tests/numbers.rs
expression: "\npub fn foo() {\n  let i = 100_000\n  let f = 100_000.00101\n  f\n}\n"
---
module rec my.``mod``
let foo (): float = begin
    let i = 100_000
    let f = 100_000.00101
    f
end
